src/query/anonymization.c

```diff
@@ -574,6 +574,15 @@ typedef struct CollectMaterialContext
   char material[MAX_SEED_MATERIAL_SIZE];
 } CollectMaterialContext;
 
+static void normalize_function_name(char *func_name)
+{
+  /* TODO: add more cases. */
```

Comment:
Which are the other cases that we need to handle here and why can't they be added now?

Reply:
This is just a moved TODO which existed earlier, I only edited the wording to match the current state of code. Maybe it's ok to remove altogether?

Reply:
Yes, please remove it, it is not helpful at all.